### ISSUE TITLE
[Project 3] Question on scheduling priority

### ISSUE THREAD

**Issue by polarcoke2** [swsnu/osspr2021#78]
조교님 안녕하세요, Team 2 입니다.
Linux scheduling priority 관련 질문이 있어 이슈 남깁니다.
WRR의 run queue에서 linux의 task priority를 고려해야 하나요? rt_rq의 경우 RR policy를 사용하더라도 각 priority (0-99)에 따라 scheduling의 우선순위를 두는데, wrr도 같은 방식을 따라야 하는지 질문 드립니다.
감사합니다.

**Comment by WoosukKwon** [comment]
No. There is no priority between WRR tasks. Thus, you can maintain a single WRR run queue for each CPU.

**Comment by polarcoke2** [comment]
감사합니다!
